### ISSUE TITLE
Get-VSTeamWorkItem not working but Show-VSTeamWorkItem works for particular ID's.

### ISSUE THREAD

**Issue by cwelde** [DarqueWarrior/vsteam#123]
Steps to reproduce
------------------
Get-VSTeamWorkItem -ProjectName myProject -Id 7

Expected behavior
-----------------
I expected it to return a work item (Work item with Id #7)

Actual behavior
---------------
Page not found. However, calling Show-VSTeamWorkItem -ProjectName myProject -Id 7 works
and correctly displays the item in the web browser

Environment data
----------------

<!-- provide the output of $VSTeamVersionTable -->

```powershell
> Get-VSTeamAPIVersion
Name                           Value                                                                                                                                                                                                                             
----                           -----                                                                                                                                                                                                                             
ServiceFabricEndpoint                                                                                                                                                                                                                                            
Build                          3.0                                                                                                                                                                                                                               
ExtensionsManagement           3.0-preview                                                                                                                                                                                                                       
Release                        3.0-preview                                                                                                                                                                                                                       
Version                        TFS2017                                                                                                                                                                                                                           
MemberEntitlementManagement                                                                                                                                                                                                                                      
Packaging                                                                                                                                                                                                                                                        
Tfvc                           3.0                                                                                                                                                                                                                               
Git                            3.0                                                                                                                                                                                                                               
Core                           3.0                                                                                                                                                                                                                               
DistributedTask                3.0-preview  
```
We are running TFS 2017 Update 3
<!-- provide the output of $PSVersionTable -->

```powershell
> $PSVersionTable
PSVersion                      5.1.17134.407                                                                                                                                                                                                                     
PSEdition                      Desktop                                                                                                                                                                                                                           
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}                                                                                                                                                                                                           
BuildVersion                   10.0.17134.407                                                                                                                                                                                                                    
CLRVersion                     4.0.30319.42000                                                                                                                                                                                                                   
WSManStackVersion              3.0                                                                                                                                                                                                                               
PSRemotingProtocolVersion      2.3                                                                                                                                                                                                                               
SerializationVersion           1.1.0.1  
```


![capture](https://user-images.githubusercontent.com/28943926/49484299-5e85c380-f7eb-11e8-9272-2204798e6d96.PNG)

Added screenshot of running the command with -Verbose

**Comment by DarqueWarrior** [comment]
Looking into it now.

**Comment by DarqueWarrior** [comment]
I was able to duplicate and I see how to get it to work. I am testing fix against 2018 and Azure DevOps (AzD)

**Comment by DarqueWarrior** [comment]
>           Donovan, I was able to figure it out this morning, thank you. I had a quick question, using the VSTeam module, is it possible to get each git repository in TFS, and then verify that the master branch for those repositories have the branch protection policy option/setting ‘Cannot delete master’ set?
> We wanted to iterate through all of our git repos in TFS and make sure this option is set. (There are many of them, so doing this manually would take up a lot of time)

You can get all the repos. You would then have to use Invoke-VSTeamRequest to call the API to check for the policy because as of now it is not wrapped.

**Comment by DarqueWarrior** [comment]
Fixed in PR #124
